cmd/bundle/utils/utils.go

```diff
@@ -36,5 +39,30 @@ func ConfigureBundleWithVariables(cmd *cobra.Command) *bundle.Bundle {
 	// Initialize variables by assigning them values passed as command line flags
 	configureVariables(cmd, b, variables)
 
+	engine, err := deploymentEngine(ctx)
+	if err != nil {
+		logdiag.LogError(ctx, err)
+		return b
+	}
+	b.DirectDeployment = engine == "direct-exp"
```

Comment:
The comment from the original impl was useful, can you copy it here.

Reply:
Missed that, thanks